builder/dockerfile/builder.go

```diff
@@ -218,7 +225,7 @@ func (b *Builder) build(stdout io.Writer, stderr io.Writer, out io.Writer) (stri
 		for k, v := range b.options.Labels {
 			line += fmt.Sprintf("%q=%q ", k, v)
 		}
-		_, node, err := parser.ParseLine(line)
+		_, node, err := parser.ParseLine(line, &b.directive)
```

Comment:
Even if we skip the `Node` change, this `ParseLine` call doesn't actually need to share the directive, right? We always put together the same static string here so it can't have different results based on the directive. Even if we add some directive in the future that would behave differently, we would use default in here to avoid conflicts.